composer.json

```diff
@@ -34,7 +34,7 @@
         "propel/propel1": "~1.7",
         "symfony/yaml": "2.*",
         "symfony/translation": "~2.0",
-        "symfony/validator": "~2.0",
+        "symfony/validator": "~2.0, <2.2",
```

Comment:
Yeah, i know. but there is an incompatibility with the 2.2 version (semver was adopted with 2.3 in symfony if i am not mistaken)
In this PR i just wanted to have the tests green in the least amount of changes possible.